examples/dit/train_dit.py

```diff
@@ -0,0 +1,411 @@
+"""
+DiT training pipeline
+- Image finetuning conditioned on class labels
```

Comment:
better add comments here to explain what is the main difference from train.py and its usage